quil-rs/src/instruction/calibration.rs

```diff
@@ -104,7 +109,7 @@ mod test_measure_calibration_definition {
         insta::with_settings!({
             snapshot_suffix => description,
         }, {
-            assert_snapshot!(measure_cal_def.to_string())
+            assert_snapshot!(measure_cal_def.to_quil_or_debug())
```

Comment:
If this test suite isn't going to have a placeholder case, should this just be `to_quil()`?

Reply:
I thought this might be easier to debug - it'll just show up in the diff rather than a terminal dump with `unwrap()`